packages/protocol/deployments/mainnet-contract-logs.md

```diff
@@ -28,6 +28,8 @@
   - bridged_erc1155: `0x3c90963cfba436400b0f9c46aa9224cb379c2c40`
   - bridged_erc721: `0xc3310905e2bc9cfb198695b75ef3e5b69c6a1bf7`
   - bridged_erc20: `0x79bc0aada00fcf6e7ab514bfeb093b5fae3653e3`
+  - bridge_watchdog: `0x00000291ab79c55dc4fcd97dfba4880df4b93624`
+  - quota_manager: `0x91f67118DD47d502B1f0C354D0611997B022f29Es`
```

Comment:
please also add quota manager related info below.

Reply:
please also list all tokens and their quota in this file as well.

Reply:
Added: https://github.com/taikoxyz/taiko-mono/pull/17162/commits/8fbdf698ec07418ff6ccd911a3b5afb3801d4e3a